cmd/edit/ingress/cmd.go

```diff
@@ -218,7 +222,11 @@ func run(cmd *cobra.Command, argv []string) {
 		os.Exit(1)
 	}
 
-	if private == nil && len(routeSelectors) == 0 {
+	sameRouteSelectors := reflect.DeepEqual(curRouteSelectors, ingress.RouteSelectors()) || ingress.RouteSelectors() == nil
```

Comment:
/nitpick
I would invert the conditional check here to short-circuit it, since comparing against `nil` is cheaper than doing a `DeepEqual`.

Reply:
Definitely!